pkg/investigations/chgm/chgm.go

```diff
@@ -55,6 +55,14 @@ func InvestigateTriggered(r *investigation.Resources) error {
 		return r.PdClient.SilenceAlertWithNote(res.string() + "Unrelated limited support reason present on cluster, silenced.")
 	}
 
+	longHibernation, hibernationErr := investigateHibernation(r.Cluster, r.OcmClient)
+	if hibernationErr != nil {
+		logging.Info(fmt.Errorf("Could not check hibernation status of cluster: %w", err))
+	}
+	if longHibernation {
+		err = r.PdClient.AddNote(fmt.Sprintf("Cluster was hibernated more than %.0f days - investigate CSRs and kubelet certificates.", HIBERNATION_TOO_LONG.Hours()/24))
```

Comment:
One small suggestion: I see that we have an SOP now, could we maybe link to it in the notes since it provides the resolution steps? https://github.com/openshift/ops-sop/blob/master/v4/alerts/cluster_has_gone_missing.md#24-hibernation